code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -616,4 +616,25 @@
 	new /obj/item/weapon/grenade/chem_grenade/metalfoam(src)
 	new /obj/item/weapon/grenade/chem_grenade/metalfoam(src)
 	new /obj/item/weapon/grenade/chem_grenade/metalfoam(src)
-	new /obj/item/weapon/grenade/chem_grenade/metalfoam(src)
\ No newline at end of file
+	new /obj/item/weapon/grenade/chem_grenade/metalfoam(src)
+
+/obj/item/weapon/storage/box/bacon
+	name = "box of bacon"
+	desc = "To be used to encouage bacon usage."

```

Comment:
GitHub is being weird again, this comment was made about the box's description.
The description makes no sense to me.